### PR TITLE
[enterprise-4.14] OCPBUGS#32782: Doc improvements related to scaling storage of clusters

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.14

Issue:
[OCPBUGS-32782](https://issues.redhat.com/browse/OCPBUGS-32782)

Link to docs preview:
[Preview](https://75066--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-about-scaling-storage-of-cluster_logical-volume-manager-storage)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@jakobmoellerdev @suleymanakbas91 
Could you please review this PR?
Thanks!

File: modules/lvms-about-scaling-storage-of-clusters.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-scaling-storage-of-cluster_{context}"]
+= Ways to scale up the storage of clusters
+
+{product-title} supports additional worker nodes for {sno} clusters on bare metal user-provisioned infrastructure. You can scale up the storage of clusters either by adding new worker nodes with available storage or by adding new devices to the existing worker nodes.
```
Comment (jakobmoellerdev) [inline modules/lvms-about-scaling-storage-of-clusters.adoc:9]: I think multiple nodes are 4.15 and above only:
https://issues.redhat.com/browse/OCPSTRAT-390

That would mean we only run SNO on 4.14, not SNO+workers. @suleymanakbas91 feel free to correct me here if necessary.

File: modules/lvms-scaling-storage-of-clusters-using-web-console.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-web-console_{context}"]
+= Scaling up the storage of clusters by using the web console
+
+You can scale up the storage capacity of the worker nodes on a cluster by using the {product-title} web console.
```
Comment (jakobmoellerdev) [inline modules/lvms-scaling-storage-of-clusters-using-web-console.adoc:9]: Same comment as above

File: modules/lvms-about-scaling-storage-of-clusters.adoc (hunk 1 above)
Comment (DanielFroehlich) [reply modules/lvms-about-scaling-storage-of-clusters.adoc:9]: Confirmed: SNO+workers would be multi-node, which is **only** supported starting V415. So "by adding new worker nodes" is available only starting V4.15. In previous versions, only "by adding new devices to the existing node" is feasable with <=4.14

Comment (sr1kar99) [reply modules/lvms-about-scaling-storage-of-clusters.adoc:9]: Hi @DanielFroehlich 
Thanks for the confirmation.

I came across a link to this module "[Adding worker nodes to SNO](https://75066--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes)".

The above section has a note saying:
"Unlike multi-node clusters, by default all ingress traffic is routed to the single control-plane node, even after adding additional worker nodes."

So would this make SNO+worker nodes different from multi-node?

If "Adding additional worker nodes to SNO" is supported with <= 4.14, then would "scaling up the storage capacity of the worker nodes" also be supported on SNO if user has added additional worker nodes?

**Comment by sr1kar99** [comment]
@radeore 
Can you please review this PR?
Thanks!

File: modules/lvms-scaling-storage-of-clusters-using-web-console.adoc (hunk 2 above)
Comment (sr1kar99) [reply modules/lvms-scaling-storage-of-clusters-using-web-console.adoc]: Updated as per the suggestion.

File: modules/lvms-about-scaling-storage-of-clusters.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/lvms-about-scaling-storage-of-clusters.adoc]: Updated as per the suggestion.

**Comment by sr1kar99** [comment]
@jakobmoellerdev 
Updated the content as per the suggestions.
Could you PTAL?
Thanks!

File: modules/lvms-scaling-storage-of-clusters-using-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+.Procedure
+
+. Edit the `LVMCluster` CR by running the following command:
++
+[source, terminal]
+----
+$ oc edit <lvmcluster_file_name> -n <namespace>
+----
+
+. Add the path to the new device in the `deviceSelector` field.
```
Comment (adellape) [inline modules/lvms-scaling-storage-of-clusters-using-cli.adoc:26]:
```suggestion
. Add the path to the new device in the `deviceSelector` field:
```

**Review by adellape — COMMENTED** [review]
🔥

**Comment by adellape** [comment]
@sr1kar99 It looks like this PR is more or less the 4.14 version of the earlier https://github.com/openshift/openshift-docs/pull/71942, which merged previously to `main`, 4.15, and 4.16; can you confirm? Just want to make it clear during merge review, so that there's no questions about why this is targeting the `enterprise-4.14` branch only. Thanks!

**Comment by sr1kar99** [comment]
> @sr1kar99 It looks like this PR is more or less the 4.14 version of the earlier #71942, which merged previously to `main`, 4.15, and 4.16; can you confirm? Just want to make it clear during merge review, so that there's no questions about why this is targeting the `enterprise-4.14` branch only. Thanks!

Hi @adellape 
Yes! You are right. Similar PR for 4.15+ has been merged already. The reason for having a separate PR for 4.14 is that 4.14 and lesser versions support this Operator only single-node OpenShift.

Thanks for adding the 4.15+ PR link. :)

File: modules/lvms-scaling-storage-of-clusters-using-cli.adoc (hunk 3 above)
Comment (sr1kar99) [reply modules/lvms-scaling-storage-of-clusters-using-cli.adoc]: Updated as per the suggestion.

**Pull request merged**